src/hooks/users.ts

```diff
@@ -105,6 +108,40 @@ export const useRecentWorldUsers = (): {
   );
 };
 
+type LastSeenAt = {
+  venueName: string;
+  venueUrl: string;
+};
+
+export const useRecentWorldUser = (
+  userId?: string
+): {
+  user?: WithId<User>;
+  lastSeenAt: LastSeenAt;
+} => {
+  const { recentWorldUsers } = useRecentWorldUsers();
+
+  const user = useMemo(() => {
+    return recentWorldUsers.find((worldUser) => worldUser.id === userId);
+  }, [userId, recentWorldUsers]);
+
+  const venueName = getUserCurrentLocation(user) ?? "";
+  const venueUrl = createUrlSafeName(venueName);
+
+  const lastSeenAt: LastSeenAt = {
+    venueName,
+    venueUrl,
+  };
```

Comment:
We shouldn't 'overload' the term 'last seen at' since we explicitly use that term to relate to a different concept already (`lastSeenAt` is the time the user was last seen, regardless of venue/etc)

Since both `lastSeenIn` and `lastSeenAt` can both be read directly off the `user`, do we need to be doing anything extra here?

Reply:
definitely agree! fixed in 5a3581bc4abfb14880e2e621a12c0994e01cace8